modules/infrastructure-moving-monitoring.adoc

```diff
@@ -5,10 +5,65 @@
 [id="infrastructure-moving-monitoring-{context}"]
 = Moving the monitoring solution
 
-[IMPORTANT]
-====
-The monitoring solution uses Cluster Version Operator (CVO) to create the
-ConfigMap that the monitoring Operator uses to determine how to deploy its
-resources. Because it uses the CVO and users cannot modify the CVO, you cannot
-change where the Operator deploys resources.
-====
\ No newline at end of file
+
+By default, a Prometheus, Grafana, and AlertManager stack is deployed to
+provide cluster monitoring. It is managed by the Cluster Monitoring Operator,
+and to move its components to different machines, you apply a custom ConfigMap.
```

Comment:
I think it should be
change "stack is" to "stacks are"
"It is managed" to "they are managed"
"you apply" to "you should apply"

Reply:
It's one stack that contains multiple components, so it's right to use singular here. I'll make that clearer.